.travis/check-vendor-extra-updated.sh

```diff
@@ -1,7 +1,20 @@
 #!/bin/bash
 set -e
 
-output=$(docker-compose --file docker-compose.yaml --file docker-compose.test.yaml run app composer update libero/content-page-bundle --with-dependencies --dry-run --no-ansi)
+function finish {
+    docker-compose --file docker-compose.yaml --file docker-compose.test.yaml down --volumes
+}
+
+trap finish EXIT
+
+cd "$(dirname "$0")/.."
+
+while IFS=$'\n' read -r line; do packages+=("$line"); done < <(< composer.lock jq --raw-output '.packages[] | select(.dist.url | startswith("./vendor-extra/")) .name')
+
+output=$(docker-compose --file docker-compose.yaml --file docker-compose.test.yaml run app composer update --dry-run --no-ansi)
 echo "${output}"
 
-[[ ${output} =~ "Nothing to install or update" ]]
+for package in "${packages[@]}";
+do
+    [[ ${output} != *"Updating ${package}"* ]]
```

Comment:
The output of this does not say much, could try adding `set -x` at the start of the script? Or making this the condition an explicit `if` that prints an error message before `exit $someNumber`